### ISSUE TITLE
fluentd dies with an error

### ISSUE THREAD

**Issue by scripter-v** [kazegusuri/fluent-plugin-prometheus#1]
In my case fluentd can't start when enabled prometheus filter.
Config:
```
<source>
  type unix
  path td-agent.sock
</source>

<source>
  type prometheus
</source>

<source>
  type tail
  format nginx
  tag nginx
  path /var/log/nginx/access.log
  pos_file fluent_nginx.pos
  types size:integer
</source>

<filter nginx>
  type prometheus

  # You can use counter type with specifying a key,
  # and increments counter by the value
  <metric>
    name nginx_size_counter_bytes
    type counter
    desc nginx bytes sent
    key size
    <labels>
       host ${hostname}
    </labels>
  </metric>

  # You can use counter type without specifying a key
  # This just increments counter by 1
  <metric>
    name nginx_record_counts
    type counter
    desc the number of emited records
    <labels>
       host ${hostname}
    </labels>
  </metric>
</filter>

<system>
  log_level debug
</system>
```

The error message from log:
```
[scripter@dev-main:~]$ fluentd -c fluent.conf
2016-01-14 19:19:36 +0300 [info]: reading config file path="fluent.conf"
2016-01-14 19:19:36 +0300 [info]: starting fluentd-0.12.19
2016-01-14 19:19:36 +0300 [info]: gem 'fluent-plugin-influxdb' version '0.2.2'
2016-01-14 19:19:36 +0300 [info]: gem 'fluent-plugin-prometheus' version '0.1.0'
2016-01-14 19:19:36 +0300 [info]: gem 'fluentd' version '0.12.19'
2016-01-14 19:19:36 +0300 [info]: adding match pattern="message" type="copy"
2016-01-14 19:19:36 +0300 [debug]: adding store type="prometheus"
2016-01-14 19:19:36 +0300 [error]: unexpected error error="undefined method `[]' for #<Fluent::Log:0x00000002362a20>"
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/plugin/filter_record_transformer.rb:174:in `initialize'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluent-plugin-prometheus-0.1.0/lib/fluent/plugin/prometheus.rb:49:in `new'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluent-plugin-prometheus-0.1.0/lib/fluent/plugin/prometheus.rb:49:in `placeholder_expnader'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluent-plugin-prometheus-0.1.0/lib/fluent/plugin/prometheus.rb:65:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluent-plugin-prometheus-0.1.0/lib/fluent/plugin/out_prometheus.rb:14:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/plugin/out_copy.rb:44:in `block in configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/plugin/out_copy.rb:33:in `each'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/plugin/out_copy.rb:33:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/agent.rb:129:in `add_match'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/agent.rb:60:in `block in configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/agent.rb:54:in `each'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/agent.rb:54:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/root_agent.rb:82:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/engine.rb:117:in `configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/engine.rb:91:in `run_configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:515:in `run_configure'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:146:in `block in start'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:352:in `call'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:352:in `main_process'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:325:in `block in supervise'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:324:in `fork'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:324:in `supervise'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/supervisor.rb:142:in `start'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/lib/fluent/command/fluentd.rb:171:in `<top (required)>'
  2016-01-14 19:19:36 +0300 [error]: /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
  2016-01-14 19:19:36 +0300 [error]: /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
  2016-01-14 19:19:36 +0300 [error]: /var/lib/gems/1.9.1/gems/fluentd-0.12.19/bin/fluentd:6:in `<top (required)>'
  2016-01-14 19:19:36 +0300 [error]: /usr/local/bin/fluentd:23:in `load'
  2016-01-14 19:19:36 +0300 [error]: /usr/local/bin/fluentd:23:in `<main>'
2016-01-14 19:19:36 +0300 [info]: process finished code=256
2016-01-14 19:19:36 +0300 [warn]: process died within 1 second. exit.
```
Seems like there is an error in 49 and 56 lines of ```prometheus.rb```. Class ```PlaceholderExpander``` has two parameters according to declaration in ```filter_record_transformer.rb```, but in ```prometheus.rb``` constructor of ```PlaceholderExpander``` is calling with only one parameter (```log```).

I tried to replace ```new(log)``` on ```new({ :log => log })``` and everything works well after that. I'm not a ruby-professional and not a fluent-expert and I don't fully understand what I did, so I kindly ask you to check it out :)

**Comment by juntagabor** [comment]
+1 

Ran into this issue too.

**Comment by kazegusuri** [comment]
thanks for filing issue. will check it later.

**Comment by kazegusuri** [comment]
Fixed this issue and released `fluent-plugin-prometheus` `0.1.1`, please try it.
Feel free to reopen if there are any problems.
